### ISSUE TITLE
how many epochs did you train?

### ISSUE THREAD

**Issue by liulai** [amanchadha/iSeeBetter#9]

**Comment by amanchadha** [comment]
We fine-tuned RBPN for four epochs and experimented with both L2 loss and APIT loss.
